plugins/scaffolder-backend/src/service/router.ts

```diff
@@ -67,7 +67,6 @@ export interface RouterOptions {
   scheduler?: PluginTaskScheduler;
 
   actions?: TemplateAction<any>[];
-  taskWorkers?: number;
```

Comment:
We can't remove this since that's a breaking change. What I'm thinking we do instead is that we expose `concurrentTasksLimit` as an option here as well, and we make the two synonymous. Basically this:

```ts
const concurrentTasksLimit = options.concurrentTasksLimit ?? options.taskWorkers
```

We then deprecate `taskWorkers`, so that we can remove it once we hit 2.0 of `scaffolder-backend`

Reply:
I added taskWorkers option back.